src/server/auth/cmds/cmds_test.go

```diff
@@ -45,8 +45,8 @@ func activateAuth(t *testing.T) {
 	// TODO(msteffen): Make sure client & server have the same version
 	// Logout (to clear any expired tokens) and activate Pachyderm auth
 	require.NoError(t, tu.Cmd("pachctl", "auth", "logout").Run())
-	cmd := tu.Cmd("pachctl", "auth", "activate")
-	cmd.Stdin = strings.NewReader("admin\n")
+	cmd := tu.Cmd("pachctl", "auth", "activate", "--supply-root-token")
+	cmd.Stdin = strings.NewReader("iamroot\n")
```

Comment:
would it make sense to do `tu.RootToken + "\n"` (or maybe plain `tu.RootToken` would work?) in case we change it for some reason?